### ISSUE TITLE
Testing layout in RPi - issues

### ISSUE THREAD

**Issue by sergiohf** [pcca-matrix/PCCA-Layout#8]
I have tried your layout on my RPi 3. Although your layout is made for windows, I think it can be more universal, since AttractMode is the closest an RPi owner will get to Hyperspin. I have recorded a video of the layout in my system. Basically animations work (which is great), but with no background. The animations are a little slow, but i think if you could speed up the animations a little bit will be perfect.  As for the background i think the layout could have a more simple approach (less animations or so) to become more compatible with the Pi. Or have a Pi version, more simpler. Congratulations once again for your amazing layout.
Here is the video: https://youtu.be/QcddkaPP_zs

**Comment by pcca-matrix** [comment]
yes, the pi does not use shaders in the same way, the problem is that half of the animations are made using glsl shaders.
I will see for a lighter version in the future.

**Comment by sergiohf** [comment]
Thanks. The RPi 4 has opengl 3.0.0, so i think maybe is compatible with the new Pi. Since i have the RPi 3, i dont think worth develop for a board that is outdated. But thanks anyway

**Comment by sergiohf** [comment]
I am still really impressed with the animations that are working in the RPi 3. I am trying, but my knowledge dont allow me to simplify the layout to remove the glsl references and replace then only to load the theme arts and working animations. 
If could ask, only when/if you get the time and disposition, to only remove the glsl references, to the theme load (like the hyperspin.nut) but with all these animations, would be great.

**Comment by pcca-matrix** [comment]
light version for the pi is planned
